cc.c

```diff
@@ -233,8 +206,36 @@ int main(int argc, char** argv)
 		}
 		else
 		{
-			fputs("UNKNOWN ARGUMENT\n", stdout);
-			exit(EXIT_FAILURE);
+			if(match(argv[i],"-f")||match(argv[i],"--file"))
```

Comment:
Your whitespaces are all messed up here and on other lines

Reply:
Oops. Should be fixed now. I've also got a PR for [M2-Mesoplanet](https://github.com/oriansj/M2-Mesoplanet/pull/9) in case you weren't notified.